awx/main/conf.py

```diff
@@ -290,6 +290,16 @@ def _load_default_license_from_file():
     placeholder={'HTTP_PROXY': 'myproxy.local:8080'},
 )
 
+register(
```

Comment:
Where is this flag actually *enforced*?  Did you mean to update this?

https://github.com/ansible/awx/pull/3284/files#diff-9d4ea1dd908b35fb92eaede4bd10bb46R326

Reply:
No, we enforce it on the `INSIGHTS_DATA_ENABLED` setting, not sure how that got lost.